### PR TITLE
changefeed: add and refine alerts

### PR DESCRIPTION
**Pull request by grovecai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qqqdan — APPROVED** [review]

File: tidb-cloud/monitor-built-in-alerting.md [hunk 1]
```diff
@@ -109,6 +109,8 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Changefeed status is failed                 | Please check status of the Changefeed in ListPage and DetailPage of TiDB Cloud Console, there may be some error messages can help you diagnose what is happening. <br> If the problem can not be fixed from your side, please contact our support team directly for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:115]:
```suggestion
| The Changefeed state is `FAILED`.                 | Check the changefeed state in the ListPage and DetailPage of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 2]
```diff
@@ -109,6 +109,8 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Changefeed status is warning                | Please check status of the Changefeed in ListPage and DetailPage of TiDB Cloud Console, there may be some error messages can help you diagnose what is happening. <br> If the problem can not be fixed from your side, please contact our support team directly for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:116]:
```suggestion
| The changefeed state is  `WARNING`.              | Check the changefeed state in the ListPage and DetailPage of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 3]
```diff
@@ -109,6 +109,8 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| Changefeed latency is more than 600 seconds | Please check status of the Changefeed in ListPage and DetailPage of TiDB Cloud Console, there may be some error messages can help you diagnose what is happening. <br> There are several reasons can trigger this alarm. <br> 1. The overall traffic of upstream has increased, and the existing Changefeed specification is insufficient to handle it. If it is temporary traffic, it will automatically recover after a period of time. If it is continuous traffic, please scale up the Changefeed. <br> 2. Downstream and network abnormal, if it is this problem, please correct it first. <br> 3. Tables lack of indexes if downstream is RDS, which may cause low write performance and high latency, please add necessary indexes to upstream or downstream. <br> If the problem can not be fixed, please contact our support team directly for further assistance.   |
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:114]:
```suggestion
| The changefeed latency exceeds 600 seconds. | Check the changefeed state in the ListPage and DetailPage of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br> Possible reasons that can trigger this alter include: <br> 1. The overall traffic in the upstream has increased, causing the existing changefeed specification to be insufficient to handle it. If the traffic increase is temporary, the changefeed latency will automatically recover after the traffic returns to normal. If the traffic increase is continuous, you need to scale up the changefeed. <br> 2. The downstream or network is abnormal. In this case, resolve this abnormality first. <br> 3. Tables lack indexes if the downstream is RDS, which might cause low write performance and high latency. In this case, you need to add the necessary indexes to the upstream or downstream. <br> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.   |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 4]
```diff
@@ -109,6 +109,8 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| The changefeed status is  `WARNING`.              | Check the changefeed status in the Changefeed Page and Changefeed Detail Page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:116]:
```suggestion
| The changefeed latency exceeds 600 seconds. | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br> Possible reasons that can trigger this alert include: <br/> - The overall traffic in the upstream has increased, causing the existing changefeed specification to be insufficient to handle it. If the traffic increase is temporary, the changefeed latency will automatically recover after the traffic returns to normal. If the traffic increase is continuous, you need to scale up the changefeed. <br/> - The downstream or network is abnormal. In this case, resolve this abnormality first. <br/> - Tables lack indexes if the downstream is RDS, which might cause low write performance and high latency. In this case, you need to add the necessary indexes to the upstream or downstream. <br> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance. |
| The changefeed status is `FAILED`.                 | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
| The changefeed status is  `WARNING`.              | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 5]
```diff
@@ -109,6 +109,8 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| The changefeed latency exceeds 600 seconds. | Check the changefeed status in the Changefeed Page and Changefeed Detail Page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br> Possible reasons that can trigger this alert include: <br/> - The overall traffic in the upstream has increased, causing the existing changefeed specification to be insufficient to handle it. If the traffic increase is temporary, the changefeed latency will automatically recover after the traffic returns to normal. If the traffic increase is continuous, you need to scale up the changefeed. <br/> - The downstream or network is abnormal. In this case, resolve this abnormality first. <br/> - Tables lack indexes if the downstream is RDS, which might cause low write performance and high latency. In this case, you need to add the necessary indexes to the upstream or downstream. <br> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance. |
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:114]:
```suggestion
| The changefeed latency exceeds 600 seconds. | Check the changefeed statuson the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br> Possible reasons that can trigger this alert include: <br/> - The overall traffic in the upstream has increased, causing the existing changefeed specification to be insufficient to handle it. If the traffic increase is temporary, the changefeed latency will automatically recover after the traffic returns to normal. If the traffic increase is continuous, you need to scale up the changefeed. <br/> - The downstream or network is abnormal. In this case, resolve this abnormality first. <br/> - Tables lack indexes if the downstream is RDS, which might cause low write performance and high latency. In this case, you need to add the necessary indexes to the upstream or downstream. <br> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance. |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 6]
```diff
@@ -109,6 +109,8 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| The changefeed status is `FAILED`.                 | Check the changefeed status in the Changefeed Page and Changefeed Detail Page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:115]:
```suggestion
| The changefeed status is `FAILED`.                 | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

File: tidb-cloud/monitor-built-in-alerting.md (hunk 4 above)
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:116]:
```suggestion
| The changefeed status is  `WARNING`.              | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 7]
```diff
@@ -109,6 +109,8 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| The changefeed status is `WARNING`.              | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (qiancai) [inline tidb-cloud/monitor-built-in-alerting.md:116]:
```suggestion
| The changefeed latency exceeds 600 seconds. | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br> Possible reasons that can trigger this alert include: <br/> - The overall traffic in the upstream has increased, causing the existing changefeed specification to be insufficient to handle it. If the traffic increase is temporary, the changefeed latency will automatically recover after the traffic returns to normal. If the traffic increase is continuous, you need to scale up the changefeed. <br/> - The downstream or network is abnormal. In this case, resolve this abnormality first. <br/> - Tables lack indexes if the downstream is RDS, which might cause low write performance and high latency. In this case, you need to add the necessary indexes to the upstream or downstream. <br> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance. |
| The changefeed status is `FAILED`.                 | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> If the problem cannot be fixed from your side, you can contact TiDB Cloud Support for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
| The changefeed status is `WARNING`.              | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> If the problem cannot be fixed from your side, you can contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for further assistance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

File: tidb-cloud/monitor-built-in-alerting.md [hunk 8]
```diff
@@ -109,6 +109,8 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| The changefeed latency exceeds 600 seconds. | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br> Possible reasons that can trigger this alert include: <br/> - The overall traffic in the upstream has increased, causing the existing changefeed specification to be insufficient to handle it. If the traffic increase is temporary, the changefeed latency will automatically recover after the traffic returns to normal. If the traffic increase is continuous, you need to scale up the changefeed. <br/> - The downstream or network is abnormal. In this case, resolve this abnormality first. <br/> - Tables lack indexes if the downstream is RDS, which might cause low write performance and high latency. In this case, you need to add the necessary indexes to the upstream or downstream. <br> If the problem cannot be fixed from your side, you can contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for further assistance. |
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:114]:
```suggestion
| The changefeed latency exceeds 600 seconds. | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> Possible reasons that can trigger this alert include: <br/> - The overall traffic in the upstream has increased, causing the existing changefeed specification to be insufficient to handle it. If the traffic increase is temporary, the changefeed latency will automatically recover after the traffic returns to normal. If the traffic increase is continuous, you need to scale up the changefeed. <br/> - The downstream or network is abnormal. In this case, resolve this abnormality first. <br/> - Tables lack indexes if the downstream is RDS, which might cause low write performance and high latency. In this case, you need to add the necessary indexes to the upstream or downstream. <br/> If the problem cannot be fixed from your side, you can contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for further assistance. |
```

Comment (ran-huang) [reply tidb-cloud/monitor-built-in-alerting.md:114]: the list should be written in html tags `<ul><li></li></ul>`

File: tidb-cloud/monitor-built-in-alerting.md [hunk 9]
```diff
@@ -109,6 +109,8 @@ The following table provides the TiDB Cloud built-in alert conditions and the co
...
+| The changefeed latency exceeds 600 seconds. | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> Possible reasons that can trigger this alert include: <br/> - The overall traffic in the upstream has increased, causing the existing changefeed specification to be insufficient to handle it. If the traffic increase is temporary, the changefeed latency will automatically recover after the traffic returns to normal. If the traffic increase is continuous, you need to scale up the changefeed. <br/> - The downstream or network is abnormal. In this case, resolve this abnormality first. <br/> - Tables lack indexes if the downstream is RDS, which might cause low write performance and high latency. In this case, you need to add the necessary indexes to the upstream or downstream. <br/> If the problem cannot be fixed from your side, you can contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for further assistance. |
```
Comment (ran-huang) [inline tidb-cloud/monitor-built-in-alerting.md:114]:
```suggestion
| The changefeed latency exceeds 600 seconds. | Check the changefeed status on the **Changefeed** page and **Changefeed Detail** page of the TiDB Cloud console, where you can find some error messages to help diagnose this issue. <br/> Possible reasons that can trigger this alert include:<ul><li>The overall traffic in the upstream has increased, causing the existing changefeed specification to be insufficient to handle it. If the traffic increase is temporary, the changefeed latency will automatically recover after the traffic returns to normal. If the traffic increase is continuous, you need to scale up the changefeed.</li><li>The downstream or network is abnormal. In this case, resolve this abnormality first.</li><li>Tables lack indexes if the downstream is RDS, which might cause low write performance and high latency. In this case, you need to add the necessary indexes to the upstream or downstream.</li></ul>If the problem cannot be fixed from your side, you can contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for further assistance. |
```

**Pull request merged**